Description: Enable support for shared fortran libraries.
Author: Johan Van de Wauw <johan.vandewauw@gmail.com>
Forwarded: not-needed

--- a/configure.ac
+++ b/configure.ac
@@ -332,15 +332,16 @@ AC_SUBST([BUILD_FORTRAN])
 ## Build static libraries by default. Furthermore, fortran shared libraries
 ## are unsupported. Disallow a user from enabling both shared libraries and
 ## fortran.
-if test "X${enable_shared}" != "Xyes"; then
-    enable_shared="no"
-fi
-
-if test "X${enable_shared}" = "Xyes"; then
-    if test "X${BUILD_FORTRAN}" = "Xyes"; then
-        AC_MSG_ERROR([Cannot build shared fortran libraries. Please configure with --disable-fortran flag.])
-    fi
-fi
+#this test is overwritten in debian - we allow shared fortran libraries
+#if test "X${enable_shared}" != "Xyes"; then
+#    enable_shared="no"
+#fi
+#
+#if test "X${enable_shared}" = "Xyes"; then
+#    if test "X${BUILD_FORTRAN}" = "Xyes"; then
+#        AC_MSG_ERROR([Cannot build shared fortran libraries. Please configure with --disable-fortran flag.])
+#    fi
+#fi
 
 ## ----------------------------------------------------------------------
 ## Check if they would like the Java native interface (JNI) compiled
